manager/manager.go

```diff
@@ -955,6 +955,16 @@ func (m *Manager) becomeLeader(ctx context.Context) {
 					log.G(ctx).WithError(err).Error("failed to create predefined network " + p.Name)
 				}
 			}
+		} else {
+			// If it is already part of cluster, check if the store contains predefined
+			//networks like bridge/host and add it if not present.
+			for _, p := range allocator.PredefinedNetworks() {
+				if store.GetNetwork(tx, p.Name) == nil {
+					if err := store.CreateNetwork(tx, newPredefinedNetwork(p.Name, p.Driver)); err != nil {
+						log.G(ctx).WithError(err).Error("failed to create predefined network " + p.Name)
+					}
+				}
+			}
```

Comment:
Rather than duplicate this code, let's move the creation of predefined networks outside the if/else.